results/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./resultstyle.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <div>
+            <h1>
+              How'd you do, trainer?
+            </h1>
+        </div>
+        <div>
+            <button id='home-button'>catch more</button>
+        </div>
+        
+    </header>
+
+    <main>
+        <div>
+            <canvas id="pokeEncounters" width="400" height="400"></canvas>
+        </div>
+        <div>
+            <canvas id="pokesCaught" width="400" height="400"></canvas>
+        </div>
+    </main>
+
+    <footer>
+    </footer>
+    <script type="module" src="./chart.js"></script>
+    <script type="module" src="./app.js"></script>
```

Comment:
Normally you use one javascript file for an html file, I would move what you have in app.js to chart.js